### DISCUSSION TITLE
Question(s)

### DISCUSSION THREAD

**Discussion by Fireworkstars46** [BTW-Community/BTW-Public#323] [Support] [answered]
Is this mod available for 1.6.4?

**Comment by ammoore00** [comment, marked as answer]
Not yet, but we're working on it :)

**Comment by Fireworkstars46** [comment]
And this mod can make trap doors floating right?

**Reply by Fireworkstars46** [reply]
Anyone?

**Comment by Qwanticitera** [comment]
i played a lot this mod and its the difficulty i was searching, but the version its tooooooo old please can you update to at least 1.16.5 i know the gigantic effort, but really needed that. or at least 1.9 . Hope admin read this.......

**Reply by Hirachosan** [reply]
This mod will eventually update to 1.8.9 but no further. We only update for technical development reasons, this mod isn't made to be compatible with other mods and newer minecraft features make things harder and harder to balance. 

We do however plan on releasing features of this mod as seperate tiny mods for modern versions eventually though
